src/View/RouteNotFoundStrategy.php

```diff
@@ -319,7 +319,7 @@ protected function getConsoleUsage(
                  * array('--param', '--explanation')
                  */
                 if (is_array($b)) {
-                    if ((count($b) !== $tableCols || $tableType !== 2) && $table !== false) {
+                    if ((($count = count($b)) !== $tableCols || $tableType !== 2) && $table !== false) {
```

Comment:
I would prefer to have it before if than here. It will be more readable imho.

Reply:
done ;)